# 0x05-python-exceptions

## Table of  content

| Projects | Description |
|----------| ------------|
|[0-safe_print_list](0-safe_print_list.py) | python function that prints x elements of a list
|[1-safe_print_integer.py](1-safe_print_integer.py) | python function that prints an integer with "{:d}".format()
|[2-safe_print_list_integers.py](2-safe_print_list_integers.py) | python function that prints the first x elements of a list and only integers
|[3-safe_print_division.py](3-safe_print_division.py) | python function that divides 2 integers and prints the result
|[4-list_division.py](4-list_division.py) | python funciton that divides element by element 2 lists
|[5-raise_exception.py](5-raise_exception.py) | python function that raises a type exception
|[6-raise_exception_msg.py](6-raise_exception_msg.py) | python function that raises a name exception with a message
|[100-safe_print_integer_err.py](100-safe_print_integer_err.py) | python function that prints integer(advanced)
|[101-safe_function.py](101-safe_function.py) | python function that executes a function safely(advanced)
|[102-magic_calculation.py](102-magic_calculation.py) | python funciton found from its Bytecode (advanced)
|[103-python.c](103-python.c) | c functions that print some basic info about Python lists, Python bytes an Python float objects (advanced) 